doc/integrations/redshift.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d22b2168-17c7-4a77-89d1-b928f3b7d41c",
+   "metadata": {},
+   "source": [
+    "# Redshift\n",
+    "\n",
+    "This tutorial will show you how to use JupySQL with Redshift.\n",
```

Comment:
There can be a brief description about what does Redshift does / add link to [docs](https://aws.amazon.com/redshift/)

Reply:
done